docs/api/qiskit-ibm-provider/ibm_provider.md

```diff
@@ -49,6 +49,7 @@ logging.getLogger('qiskit_ibm_provider').setLevel(logging.WARNING)
 | [`IBMProvider`](qiskit_ibm_provider.IBMProvider "qiskit_ibm_provider.IBMProvider")(\[token, url, name, instance, ...]) | Provides access to the IBM Quantum services available to an account. |
 | [`IBMBackend`](qiskit_ibm_provider.IBMBackend "qiskit_ibm_provider.IBMBackend")(configuration, provider, api\_client)  | Backend class interfacing with an IBM Quantum device.                |
 | [`IBMBackendService`](qiskit_ibm_provider.IBMBackendService "qiskit_ibm_provider.IBMBackendService")(provider, hgp)    | Backend namespace for an IBM Quantum account.                        |
+| [`Session`](qiskit_ibm_provider.Session "qiskit_ibm_provider.Session")(\[max\_time])                                   | Class for creating a flexible Qiskit Runtime session.                |
```

Comment:
FYI the `"qiskit_ibm_provider.Session"` inside the  parantheses is really fancy markdown syntax for setting a title for the link, which is what shows when you hover over the link, like this:

<img width="313" alt="Screenshot 2023-10-18 at 5 12 36 PM" src="https://github.com/Qiskit/documentation/assets/14852634/850c7d37-e39a-41a7-afe8-84d5366f754f">

I thought it was a bug at first hah.